chain/chain/src/test_utils.rs

```diff
@@ -1417,7 +1417,7 @@ impl ChainGenesis {
         ChainGenesis {
             time: Clock::utc(),
             height: 0,
-            gas_limit: 1_000_000,
+            gas_limit: 10u64.pow(15),
```

Comment:
this seems to be impacting many tests.
Maybe add a comment on why did you select such value for gas limit.